src/sentry_plugins/segment/plugin.py

```diff
@@ -5,16 +5,31 @@
 
 from sentry_plugins.base import CorePluginMixin
 from sentry_plugins.utils import get_secret_field_config
+from sentry.integrations import FeatureDescription, IntegrationFeatures
+
+DESCRIPTION = """
+Send Sentry events to Segment.
+
+Segment is a customer data platform (CDP) that helps you collect, clean, and control your customer data.
```

Comment:
@jonesphillip We should probably mention that the Segment integration is a replacement to having to set up the Sentry SDK on your pages and services.